src/core/rtkQuery/splitV3Api/filterApi.ts

```diff
@@ -0,0 +1,60 @@
+import { defaultSerializeQueryArgs } from '@reduxjs/toolkit/query';
+import { omit } from 'lodash';
+
+import { splitV3Api } from '@/core/rtkQuery/splitV3Api';
+
+import type { InfiniteResultType, ListResultType, PaginationType } from '@/core/types/api';
+import type { CollectionGroupType } from '@/core/types/api/collection';
+import type { FilterType } from '@/core/types/api/filter';
+
+const filterApi = splitV3Api.injectEndpoints({
+  endpoints: build => ({
+    getFilter: build.query<FilterType, { filterId: string }>({
+      query: ({ filterId }) => ({
+        url: `Filter/${filterId}`,
+        params: {
+          withConditions: true,
+        },
+      }),
+    }),
+    getFilteredGroups: build.query<
```

Comment:
Maybe we should have some special suffix or prefix for endpoints that merge data? like Infinite